src/WorkerFarm.js

```diff
@@ -2,8 +2,10 @@ const {EventEmitter} = require('events');
 const os = require('os');
 const Farm = require('worker-farm/lib/farm');
 const promisify = require('./utils/promisify');
+const path = require('path');
 
 let shared = null;
+const SYNC_TRESHOLD = 50000; // Treshold to start workers in bytes
```

Comment:
Where did this number come from? 0.05 Megabyte doesn't seem like a very "big" project. Adding just a single image to the project would cross the threshold.

Reply:
That's kind of an issue with this approach it counts in images, i'll probably pivot towards counting for this reason and maybe performance. According to my benchmarks this is near the treshold of where things start to get quicker but i just noticed it doesn't take node_modules into consideration so...